### PR TITLE
community: Update UC toolkit documentation to use LangGraph APIs, and fix pydantic error

### PR DESCRIPTION
**Pull request by smurching**
- **Description:** Update UC toolkit documentation to show an example of using recommended LangGraph agent APIs before the existing LangChain AgentExecutor example, and fixes a `pydantic` error message due to a missing type definition in the example notebook. Tested by manually running the updated example notebook 
- **Dependencies:** No new dependencies

### CODE REVIEW

File: libs/community/langchain_community/tools/databricks/tool.py [hunk 1]
```diff
@@ -9,8 +9,9 @@
 from langchain_core.tools.base import BaseToolkit
 from typing_extensions import Self
 
+from databricks.sdk import WorkspaceClient
```
Comment (smurching) [inline libs/community/langchain_community/tools/databricks/tool.py:12]: This seems necessary to fix a pydantic forward reference issue; without it, calling `UCFunctionToolkit.include(...)` throws with


```
PydanticUserError: UCFunctionToolkit is not fully defined; you should define WorkspaceClient, then call UCFunctionToolkit.model_rebuild().
```

File: docs/docs/integrations/tools/databricks.ipynb [hunk 2]
```diff
@@ -54,11 +62,12 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
+   "execution_count": 9,
    "metadata": {},
    "outputs": [],
    "source": [
     "from langchain_community.tools.databricks import UCFunctionToolkit\n",
+    "from databricks.sdk import WorkspaceClient\n",
```
Comment (smurching) [inline docs/docs/integrations/tools/databricks.ipynb:70]: We should be able to remove this import after the change in `tool.py` below is released

**Review by efriis — COMMENTED** [review]
Hey there! I think the code change would be more relevant in the `langchain-databricks` integration here, rather than the deprecated community one:

https://github.com/langchain-ai/langchain-databricks

**Comment by efriis** [comment]
Closing because this import format isn't compatible with community's optional parameters, and would welcome this class in the langchain-databricks package, and we can deprecate this one!

**Pull request closed without merging**